### ISSUE TITLE
Change the name?

### ISSUE THREAD

**Issue by amaatouq** [amaatouq/netwise#20]
tldr: the current possible names we have are: `netwise`, `swift`, `tajriba`, `empeiria`, `empirica.ly ` ,`empirica`,  `Aravi`, `Arav`, `Aricenna`, `socioScope`, `SHubble` 
-----------

This is the [googledoc that explains the context and contains these pieces of information](https://docs.google.com/document/d/1EcR73bX1SU59otpL1rDD4lisq45KFb60BJbCqnzjGXs/edit?usp=sharing)

We might want to change the name of `netwise` as our framework is not overly optimized for networked experiments (on purpose) and it is a more general purpose experimentation platform. Therefore, these are some suggested names:

### Some general ideas for inspiration: 

1. Something related to **telescopes**: This is inspired by a quote by Duncan Watts (one of the most important Computational Social Scientists): 

    - > “The internet will be to sociology what the telescope was to physics. This device that makes the previously invisible visible and in so doing drives all types of new scientific discoveries” – Duncan Watts
    - This is a [list of names of important telescopes](https://en.wikipedia.org/wiki/List_of_space_telescopes).

2. Something related to the **history of the scientific method**:
   - The idea of empirical started with `Aristotle's` concept of `empeiria`. Then Ibn Al-Haytham ("`Alhazen`") and Ibn-Sina ("`Avincenna`") took this concept (and Avicenna called it `Tajriba`)  and developed it into the foundation of what we call today: the scientific method.
   - A passage discussing Avicenna's `tajriba` from "The Book of Healing": 
       - > In the Al-Burhan (On Demonstration) section of the book, Avicenna discussed the philosophy of science and described an early scientific method of inquiry. He discusses Aristotle’s Posterior Analytics and significantly diverged from it on several points. Avicenna discussed the issue of a proper methodology for scientific inquiry and the question of “How does one acquire the first principles of a science?” He asked how a scientist would arrive at “the initial axioms or hypotheses of a deductive science without inferring them from some more basic premises?” He explains that the ideal situation is when one grasps that a “relation holds between the terms, which would allow for absolute, universal certainty.” Avicenna then adds two further methods for arriving at the first principles: the ancient Aristotelian method of induction (istiqra), and the method of examination and experimentation (**`tajriba`**). Avicenna criticized Aristotelian induction, arguing that “it does not lead to the absolute, universal, and certain premises that it purports to provide.” In its place, he develops a “method of experimentation as a means for scientific inquiry.”
   - Another passage discussing Avicenna's `tajriba` as a way to get at knowledge: 
       - > As Avicenna describes it, tajriba  ("experience") involves the repeated sensation of some phenomenon that is preserved in memory, e.g., that purging of bile follows ingestion of scammony. It also involves a “hidden” or implicit process of reasoning. We infer that the repeated connection involves an essential relation between two things: e.g., it is not accidental that the purging of bile follows the ingestion of scammony, for regularities cannot be due to chance. This implicit process of reasoning removes doubt. So, experience gives rise to certitude. Avicenna defines certitude in terms of second-order belief: one is certain when one knows that what one has assented to cannot be otherwise. Certitude is not wholly subjective since knowledge of a proposition requires its truth. On the other hand, though experience gives rise to certitude, it does not yield knowledge in the strict sense. For knowledge in the strict sense is both necessary and explanatory. From experience, we derive our certitude that members of a certain species cause something, but experience does not show why.

3. Keep `NetWise`:
   - The current name that we have.

4. **ADD HERE ADDITIONAL IDEAS** (like the [Mayan Cave observatory](https://www.google.com/search?q=Mayan+Cave+observatory&safe=strict&tbm=isch&tbs=rimg:CZ4tvu7l4vz1Ijg9aGKOCQowAQYAblVetFOBjJaPKIbGL5J02L0rskNwZns3B-ItPAdjf0HcY7oXLEk_1sEy6JF6BHCoSCT1oYo4JCjABEUI6NBQWdhrZKhIJBgBuVV60U4ERlLlIXrKslVYqEgmMlo8ohsYvkhFmtLcdWjT1IyoSCXTYvSuyQ3BmEdXgUV6Vcu9LKhIJezcH4i08B2MRKbDfQNzcHhUqEgl_1QdxjuhcsSRHezb-9qyHj8yoSCT-wTLokXoEcEaEidXWJV9bZ&tbo=u&sa=X&ved=2ahUKEwjU0PLFoPHZAhVPmeAKHakFB9wQ9C96BAgAEBs&biw=1440&bih=717&dpr=2))

### Name suggestions based on the above:

1. From the telescope analogy:
      - `SocioScope`
          - pros:
                  - It makes you see social interactions that were not observable before! 
                  - memorable for anglophones
                  - catchy
                  - domain name available
          - cons: 
                  - a bit of a tongue twister (but I don't think it is toobad)
      - [`Swift`](https://en.wikipedia.org/wiki/Neil_Gehrels_Swift_Observatory)
          - pros:
                  - It makes you see interactions that were not observable before 
                  - memorable for anglophones
                  - catchy
          - cons: 
                  - used in many places like the [programming language](https://developer.apple.com/swift/), and the [suzki car](https://en.wikipedia.org/wiki/Suzuki_Swift) 
                  - expensive domain names
2. From the History of Science:
      - `empeiria` from Aristotle (i.e., "experience")
          - pros:
                  - Great background story, content, and meaning from the point of view of what our framework is trying to achieve 
                  - available cheap domain names
                  - not many google results in the search anyway
          - cons: 
      - `empirica ` mutants of Aristotle's empeiria
          - pros:
                  - Great background story, content, and meaning from the point of view of what our framework is trying to achieve 
                  - easy for anglo's to say
                  - the domain `empirica.ly `is available (which sounds really nice in Arabic)
          - cons: 
                  - domain name not available (i.e., empirica.io is an algorithmic trading platform for cryptocurrencies)
      - `Tajriba` or `Tajarib` (singular or plural), which is the Arabic translation of the Greek “empeiria”
          - pros:
                  - Great background story, content, and meaning from the point of view of what our framework is trying to achieve 
                  - available cheap domain names
                  - not many google results in the search anyway
                  - It is my favorite (Abdullah)
          - cons: 
                  - (probably) not memorable for anglophones
                     - That might be solved with a little explanation on the origin of the word in the readme, to help lock in the meaning
                  - Maybe hard to pronounce?
                  - not sure if it is catchy like netwise
      - Combination of the names Aristotle and Avicenna (e.g., `Aravi`, `Arav`, `Aricenna`, etc)
          - pros:
                  - nice background story
                  - available cheap domain names
          - cons: 
                  - (probably) not memorable for anglophones
                  - probably not catchy

3. Keeping `NetWise`
          - pros:
                  - we already have the domain
                  - it is catchy
          - cons: 
                  - The platform is not overly optimized for networks, it is more general purpose .. this name might make it too connected to networks and/or Wisdom of the Crowd
